project-base/storefront/components/Pages/ProductDetail/ProductDetailContent.tsx

```diff
@@ -99,6 +102,15 @@ export const ProductDetailContent: FC<ProductDetailContentProps> = ({ product, f
                 />
 
                 <ProductDetailAvailabilityList ref={scrollTarget} storeAvailabilities={product.storeAvailabilities} />
+                {isLuigisBoxActive && (
+                    <Webline className="mb-6 px-0">
```

Comment:
Another case, I guess you can put simple div here.

Reply:
I actually changed some of the HTML to be better suited for the place where its used. It should make more sense now.